balancer/xds/lrs/lrs.go

```diff
@@ -113,7 +113,7 @@ func (ls *lrsStore) buildStats() []*loadreportpb.ClusterStats {
 
 	var ret []*loadreportpb.ClusterStats
 	ret = append(ret, &loadreportpb.ClusterStats{
-		ClusterName:           ls.serviceName,
```

Comment:
With this change, ls.serviceName field will be unused, can we delete it then?

Reply:
Done